tests/director/tests/application_templates_subscription_test.go

```diff
@@ -26,6 +26,12 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	RegionValue      = "eu-1"
+	DefaultSubdomain = "compass-external-services-mock-sap-mtls"
```

Comment:
Same as above comment for `eu-1`... maybe you can get this value from `global.externalServicesMock.certSecuredHost` instead of hardcoding it